azure-resources/Network/dnsZones/recommendations.yaml

```diff
@@ -0,0 +1,18 @@
+- description: Ensure Time-To-Live (TTL) is set appropriately to ensure RPOs can be met
+  aprlGuid: 7d09523b-b3c0-403e-b104-d5d46240d683
+  recommendationTypeId: null
+  recommendationControl: Disaster Recovery
+  recommendationImpact: High
+  recommendationResourceType: Microsoft.Network/dnsZones
+  recommendationMetadataState: Active
+  longDescription: |
+    Azure Private DNS allows the Time-To-Live (TTL) for records in the zone to be set to a value between 1 and 2147483647 seconds. You should ensure that the TTL for the DNS records in your DNS Zones are set appropriately to meet your RPO targets to ensure that records are not cached by another system.
```

Comment:
```suggestion
    Azure DNS allows the Time-To-Live (TTL) for records in the zone to be set to a value between 1 and 2147483647 seconds. You should ensure that the TTL for the DNS records in your DNS Zones are set appropriately to meet your RPO targets to ensure that records are not cached by another system.
```

Reply:
> 



> TTL for the DNS records

Our documentation talks about TTLs for record sets, so may want to re-word further:

"... TTL for DNS record sets... " or "...TTL for each record set..."

https://learn.microsoft.com/en-us/azure/dns/dns-zones-records#time-to-live